packages/pdf-renderer/src/lib/pdfGenerator.service.ts

```diff
@@ -60,7 +61,7 @@ export class PdfGenerator {
     private static generatePdfStreamTask: TaskFunction<GeneratePdfPageParams, Readable> = async ({ page, data }) => {
         const pdfOptions = await PdfGenerator.preparePage(page, data);
 
-        return await page.createPDFStream(pdfOptions);
+        return Readable.fromWeb((await page.createPDFStream(pdfOptions)) as ReadableStream);
```

Comment:
According to https://github.com/puppeteer/puppeteer/pull/11805#issue-2112434944 `createPDFStream` should return [ReadableStream from stream](https://nodejs.org/api/stream.html#class-streamreadable) but it returns [ReadableStream from Web Streams API](https://nodejs.org/api/webstreams.html#class-readablestream) :skull: